ot/smooth.py

```diff
@@ -26,25 +26,40 @@
 #         Remi Flamary <remi.flamary@unice.fr>
 
 """
-Smooth and Sparse Optimal Transport solvers (KL an L2 reg.)
+Smooth and Sparse Optimal Transport solvers (KL an L2 reg.) [17] and sparsity
```

Comment:
Use a shorter first line without refs for here:
![image](https://user-images.githubusercontent.com/1386873/233966292-a509f6f3-bb34-4645-be7c-37cb555fdb60.png)

Also cleanup the text for the module that is not looking good:
![image](https://user-images.githubusercontent.com/1386873/233966441-2201187f-9a33-48fa-aa04-ab1774a80c55.png)